src/components/LogView/index.js

```diff
@@ -116,14 +122,31 @@ class FullLogLine extends React.Component {
     }),
     lineRefCallback: PropTypes.func,
     toggleBookmark: PropTypes.func,
-    wrap: PropTypes.bool
+    wrap: PropTypes.bool,
+    updateSelectStartIndex: PropTypes.func,
+    updateSelectEndIndex: PropTypes.func
   };
 
   constructor(props) {
     super(props);
     this.state = {};
   }
 
+  handleMouseUp = () => {
+    let endIndex = this.props.line.lineNumber;
+    if (window.getSelection().toString() !== '') {
```

Comment:
I think this check can be achieved by verifying `selection.anchorOffset !== selection.focusOffset`